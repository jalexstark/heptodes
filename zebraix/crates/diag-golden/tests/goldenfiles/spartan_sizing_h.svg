<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 200 279.308594 L 200 20.691406 M 11.320312 150 L 388.679688 150 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.5" stroke-miterlimit="10" d="M 11.320312 198.492188 L 388.679688 198.492188 M 11.320312 246.984375 L 388.679688 246.984375 M 11.320312 101.507812 L 388.679688 101.507812 M 11.320312 53.015625 L 388.679688 53.015625 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.5" stroke-miterlimit="10" d="M 162.265625 279.308594 L 162.265625 20.691406 M 124.527344 279.308594 L 124.527344 20.691406 M 86.792969 279.308594 L 86.792969 20.691406 M 49.054688 279.308594 L 49.054688 20.691406 M 11.320312 279.308594 L 11.320312 20.691406 M 237.734375 279.308594 L 237.734375 20.691406 M 275.472656 279.308594 L 275.472656 20.691406 M 313.207031 279.308594 L 313.207031 20.691406 M 350.945312 279.308594 L 350.945312 20.691406 M 388.679688 279.308594 L 388.679688 20.691406 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 105.660156 214.65625 L 105.660156 85.34375 M 294.339844 214.65625 L 294.339844 85.34375 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.5" stroke-miterlimit="10" d="M 105.660156 85.34375 L 294.339844 214.65625 M 294.339844 85.34375 L 105.660156 214.65625 "/>
</svg>
