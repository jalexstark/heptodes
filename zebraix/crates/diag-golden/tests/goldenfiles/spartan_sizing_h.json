{
  "sizing_scheme": "Fill",
  "axes_range": [
    2.0
  ],
  "padding": [
    0.03,
    0.08
  ],
  "drawables": [
    {
      "layer": 0,
      "drawable": {
        "Lines": {
          "start": [
            [
              0.0,
              -2.0
            ],
            [
              -2.0,
              0.0
            ]
          ],
          "end": [
            [
              0.0,
              2.0
            ],
            [
              2.0,
              0.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 0,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -2.0,
              0.0
            ]
          ],
          "end": [
            [
              2.0,
              0.0
            ]
          ],
          "offsets": [
            [
              -0.0,
              -0.75
            ],
            [
              -0.0,
              -1.5
            ],
            [
              0.0,
              0.75
            ],
            [
              0.0,
              1.5
            ]
          ]
        }
      }
    },
    {
      "layer": 0,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              0.0,
              -2.0
            ]
          ],
          "end": [
            [
              0.0,
              2.0
            ]
          ],
          "offsets": [
            [
              -0.4,
              -0.0
            ],
            [
              -0.8,
              -0.0
            ],
            [
              -1.2000000000000002,
              -0.0
            ],
            [
              -1.6,
              -0.0
            ],
            [
              -2.0,
              -0.0
            ],
            [
              0.4,
              0.0
            ],
            [
              0.8,
              0.0
            ],
            [
              1.2000000000000002,
              0.0
            ],
            [
              1.6,
              0.0
            ],
            [
              2.0,
              0.0
            ]
          ]
        }
      }
    }
  ]
}
